doc/pyplots/pdf/exponential.py

```diff
@@ -0,0 +1,18 @@
+from probfit.pdf import exponential
+from probfit.plotting import draw_pdf
+import matplotlib.pyplot as plt
+
+_arg = {"lambda": 0.5}
+draw_pdf(exponential, arg=_arg, bound=(0, 5), label=str(_arg), density=False,
+         bins=100)
+
+arg = {"lambda": 1.0}
```

Comment:
Now arg is not used anymore. You only use `_arg` in the functions. And it is always the same `_arg` now in all three `draw_pdf` calls, before it was a different `arg` all three calls. Is this as intended? I suppose you want to rename all `arg` into `_arg` and then the code is fine.